.github/workflows/check-change-tags.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019-2023 Tauri Programme within The Commons Conservancy
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-License-Identifier: MIT
+
+name: check change tags
+
+on:
+  pull_request:
+    paths:
+      - '.changes/*.md'
+  push:
```

Comment:
I think we can remove the push: hook on some workflows since we're not allowing direct pushes to dev anymore

Reply:
yeah sure